### PR TITLE
Hot Fix for JWQLDB View

### LINKED ISSUE #1083: Bad pixel monitor stats table
https://jwql.stsci.edu/jwqldb/nircam_bad_pixel_stats
The bad pixel monitor stats table page crashes, and returns:

TypeError at /jwqldb/nircam_bad_pixel_stats
jwqldb_table_viewer() got an unexpected keyword argument 'tablename_param'

### PR DESCRIPTION
**Pull request by mfixstsci**
When accessing the table view from monitors, we pass the table name as a url parameter. This PR will grab the parameter from the URL.

### CODE REVIEW

**Comment by mfixstsci** [comment]
@BradleySappington or @bhilbert4 sorry to wait until Friday, but this is ready for review!

**Review by BradleySappington — COMMENTED** [review]
Are there changes that actually call this method with a second parameter?

File: jwql/website/apps/jwql/views.py
Comment (BradleySappington) [inline jwql/website/apps/jwql/views.py:558]: Are there changes that actually call this function with the second parameter?

File: jwql/website/apps/jwql/views.py
Comment (BradleySappington) [inline jwql/website/apps/jwql/views.py:579]: if you are sending a default of `None` then you dont need the if/else.  You can simply say:
```
except KeyError:
     tablename = tablename_param
```

**Review by BradleySappington — COMMENTED** [review]
A few questions before approval

File: jwql/website/apps/jwql/views.py
Comment (mfixstsci) [reply jwql/website/apps/jwql/views.py:558]: Yes, when you are coming from the monitor view, the tablename is passed as a url parameter which is passed to this view. But if you are accessing the JWQLDB view from the navbar/other means that aren't directly linked from a monitoring page, the default is set to `None`

Comment (BradleySappington) [reply jwql/website/apps/jwql/views.py:558]: Great thanks!

File: jwql/website/apps/jwql/views.py
Comment (mfixstsci) [reply jwql/website/apps/jwql/views.py:579]: I like this idea, if `tablename_param` is None, we still achieve the same goal here.

Comment (BradleySappington) [reply jwql/website/apps/jwql/views.py:579]: @mfixstsci , are you making that change or should I merge as is?

**Review by BradleySappington — APPROVED** [review]

**Pull request merged**
